<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmModelColor</class>
 <widget class="QMainWindow" name="frmModelColor">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>431</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>颜色模式设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>背景色</string>
    </property>
    <widget class="QLineEdit" name="txtGreen_background">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblProject">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSimulation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRed_background">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBlue_background">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblSimulation_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnColorBackground">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>调色板</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblColorBack">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>模型颜色</string>
    </property>
    <widget class="QLineEdit" name="txtGreen_model">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblProject_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSimulation_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRed_model">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBlue_model">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblSimulation_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnColorModel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>调色板</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblColorModel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxColorType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>材料颜色模式</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>法向颜色模式</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标准颜色模式</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
